block-storage-attacher/tests/e2e/scripts/common.sh

```diff
@@ -38,14 +39,35 @@ function setKubeConfig {
     fi
     set_issue_repo "armada-api"
 
+     bluemix_home="$BLUEMIX_HOME"
+    if [ -z ${BLUEMIX_HOME+x} ]; then
+        bluemix_home="$HOME"
+    fi
+
+
     # Get the kube config from the `bx` cli and export KUBECONFIG for
     # your current bash session
 
     cluster_name=$1
-    echo "Generating Kube Config through 'bx cs cluster-config $cluster_name --admin' and exporting KUBECONFIG"
-    configfile=$(bx cs cluster-config $cluster_name --admin | grep ^export | cut -d '=' -f 2)
-    cat $configfile
-    export KUBECONFIG=$configfile
+    cluster_id=$(ibmcloud ks clusters | awk "/$cluster_name/"'{print $2}')
+    echo "Generating Kube Config through 'ibmcloud ks  cluster config $cluster_name --admin' and exporting KUBECONFIG"
+    ibmcloud ks  cluster config --cluster $cluster_name --admin
+    kube_config_location="$bluemix_home/.bluemix/plugins/container-service/clusters/$cluster_name-$cluster_id-admin/kube-config*.yml" 
+    echo "$kube_config_location"
+    ls -l $kube_config_location
+  
+     if ls $kube_config_location 1> /dev/null 2>&1; then
```

Comment:
we dont need to do `export KUBECONFIG` step now with latest ibmcloud plugin. So remove this if-else checks. Just cluster config will download and set to current cluster.

Reply:
The exisitng framework works with KUBECONFIG envoroment varible only that is why setting this value and passing